cdn_lambda/functions/map_to_s3.py

```diff
@@ -0,0 +1,61 @@
+import json
+import logging
+import os
+from datetime import datetime
+
+import boto3
+
+LOG = logging.getLogger("map-to-s3-lambda")
+
+
+def lambda_handler(event, context):
+    _ = context
```

Comment:
What does this do?

Reply:
The context variable is unused otherwise but I believe that the handler must be able to accept it.
So I guess all this is really doing is `pylint: disable=unused-argument`.